Inventory/Model/Source/InitTypeCodeExtensionAttributes.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model\Source;
+
+use Magento\Framework\Api\ExtensionAttributesFactory;
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\Framework\DataObject;
+use Magento\Inventory\Model\ResourceModel\GetSourceTypeBySourceCode;
+
+/**
+ * Set store-pickup related source extension attributes
+ */
+class InitTypeCodeExtensionAttributes
+{
+    /**
+     * @var ExtensionAttributesFactory
+     */
+    private $extensionAttributesFactory;
+
+    /**
+     * @var GetSourceTypeBySourceCode
+     */
+    private $getSourceTypeBySourceCode;
+
+    /**
+     * @param ExtensionAttributesFactory $extensionAttributesFactory
+     * @param GetSourceTypeBySourceCode $getSourceTypeBySourceCode
+     */
+    public function __construct(
+        ExtensionAttributesFactory $extensionAttributesFactory,
+        GetSourceTypeBySourceCode $getSourceTypeBySourceCode
+    ) {
+        $this->extensionAttributesFactory = $extensionAttributesFactory;
+        $this->getSourceTypeBySourceCode = $getSourceTypeBySourceCode;
+    }
+
+    /**
+     * Set store-pickup related source extension attributes.
```

Comment:
Please remove the "store-pickup" from phpdocs :)

Reply:
Exacly, i was thinking about this too, we can save lot of process with this change.